language/src/regent/codegen.t

```diff
@@ -2956,11 +2972,12 @@ local function make_partition_projection_functor(cx, expr, loop_index, color_spa
       [value.actions];
     end)
 
-    local terra partition_functor(runtime : c.legion_runtime_t,
+    local terra partition_functor([cx.runtime],
                                   mappable : c.legion_mappable_t,
                                   idx : uint,
                                   parent : c.legion_logical_partition_t,
                                   [point])
+      var runtime = [cx.runtime]
```

Comment:
I know this shortens the diff, but it would probably make the final code cleaner if we use `[cx.runtime]` below.